test/pipboy/parser/composite_readers_test.rb

```diff
@@ -7,14 +8,25 @@ def setup
   end
 
   def test_header
+    headerBytes = StringIO.new("\xA0\x00\x00\x00\x03")
+    assert_equal([160,3], Pipboy::Parser::CompositeReaders.header(headerBytes))
   end
 
   def test_list
+    primitiveReaders = Pipboy::Parser::PrimitiveReaders
```

Comment:
I'd probably not assign this to a variable just to keep things short, and instead break up line 18.

e.x.

```ruby
assert_equal([1, 2, 3], 
              Pipboy::Parser::CompositeReaders.list(streamBytes, primitiveReaders, :uint32))
```

Or assign the result of the reader to a variable and put that in the assert:

```ruby
parsed_list = Pipboy::Parser::CompositeReaders.list(streamBytes, primitiveReaders, :uint32)
assert_equal([1, 2, 3], parsed_list)
```